vendor/github.com/containers/podman/v4/pkg/annotations/annotations.go

```diff
@@ -40,6 +40,9 @@ const (
 	// ImageRef is the container image ref annotation.
 	ImageRef = "io.kubernetes.cri-o.ImageRef"
 
+	// ImageDigests is the container image digests annotation.
+	ImageDigests = "io.kubernetes.cri-o.ImageDigests"
```

Comment:
@cri-o/cri-o-maintainers are we happy with that annotation? I have to add it to Podman and vendor it back in here afterwards.

Reply:
why does it have to be added there?

Reply:
We have the other ones there as well. Would you prefer a different location?

Reply:
not necessarily, mostly curious :)

Reply:
name LGTM